src/js/utils/html.js

```diff
@@ -0,0 +1,19 @@
+import { getBaseApiUrl } from "../api";
+/*
+  Takes an HTML string and replaces each 'src' attribute in each 'img' tag with
+  the API url prepended.
+
+  E.g. /path/to/img.jpg -> http://api.com/path/to/img.jpg
+ */
+export function parseImgTags(html) {
+  let parsedHTML = document.createElement("div");
+  parsedHTML.innerHTML = html;
+  let imgTags = parsedHTML.querySelectorAll("img");
+
+  imgTags.forEach(function (img) {
+    let src = img.getAttribute("src");
+    img.setAttribute("src", getBaseApiUrl() + src);
```

Comment:
I believe you should be able to do this:

```suggestion
    img.src = getBaseApiUrl() + src;
```

Reply:
thanks @schalkneethling - wouldn't `src` be undefined now? would need to be `img.src = getBaseApiUrl() + img.src;`

Reply:
this doesn't work because `.src` returns the full path, which in case of local it returns `http://localhost:3000/...` instead of just the relative path

see https://stackoverflow.com/a/39972078

Reply:
I've actually removed the entire `html.js` module since it turns out not to be necessary on anything except development. Tried to quickly retrofit it to render the images on development but was struggling so I've just left it for now, we can address again in future if it becomes an issue.

Reply:
Doh! Ah well, less code :)